src/main/java/net/twasi/obsremotejava/message/request/inputs/SetInputMonitorTypeRequest.java

```diff
@@ -0,0 +1,27 @@
+package net.twasi.obsremotejava.message.request.inputs;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import net.twasi.obsremotejava.model.InputMonitor;
+
+@Getter
+@ToString(callSuper = true)
+public class SetInputMonitorTypeRequest extends InputRequest {
+    private final Data requestData;
+
+    @Builder
+    private SetInputMonitorTypeRequest(String inputName, InputMonitor.Type monitorType) {
+        super(Type.SetInputMonitorType);
+
+        this.requestData = Data.builder().inputName(inputName).build();
```

Comment:
monitorType is unused in the builder call

Reply:
whoops, forgot to add it